app/src/features/authentication/signup/hooks/useSendConfirmationEmail.ts

```diff
@@ -0,0 +1,22 @@
+import { useMutation } from "@tanstack/react-query";
+import { sendEmailVerification } from "../services/apiConfirmCode";
+
+export function UseSendConfirmationEmail() {
+  const {
+    mutate: SendConfirmationCode,
+    isPending,
+    isSuccess,
+    isError,
+  } = useMutation({
+    mutationFn: sendEmailVerification,
+    onSuccess: (data) => {
```

Comment:
this returns ts error as the data is of unknown type
use an interface or tell it which type of data will arrive

Reply:
actually no data will arrive , just the status XD